core/trino-main/src/main/java/io/trino/sql/planner/planprinter/TextRenderer.java

```diff
@@ -145,6 +151,19 @@ private String printStats(PlanRepresentation plan, NodeRepresentation node)
         return output.toString();
     }
 
+    private void printMetrics(StringBuilder output, String type, Metrics metrics)
+    {
+        if (!verbose) {
+            return;
+        }
+        Map<String, Metric> sortedMap = new TreeMap(metrics.getMetrics());
+        sortedMap.forEach((name, metric) -> {
+            if (metric instanceof LongCount) {
```

Comment:
please add support for other type of metrics too

Reply:
Added support for `Histogram` but only for printing its `getTotal()` value - since I wasn't sure if/how should I print the percentiles' estimates (IIUC, `TDigest` doesn't seem to have a "natural" `toString` implementation).